add explorer udpates (#6708)
dbt-labs/docs.getdbt.com: website/docs/docs/collaborate/explore-projects.md

<Expandable alt_header="Search features">

> - **Partial keyword search** &mdash; Also referred to as fuzzy search. Explorer uses a "contains" logic to improve your search results. This means you don't need to know the exact root word of your search term.

Reviewer: Just shifting focus on what you can use in this.
```suggestion
- **Partial keyword search** &mdash; Also referred to as fuzzy search. Explorer uses a "contains" logic to improve your search results. This means you can search for partial terms without knowing the exact root word of your search term.
```

> - **Partial keyword search** &mdash; Also referred to as fuzzy search. Explorer also uses a "contains" logic to improve your search results. This means you don't need to know the exact root word of your search term.

Author: ```suggestion
- **Partial keyword search** &mdash; Also referred to as fuzzy search. Explorer uses a "contains" logic to improve your search results. This means you don't need to know the exact root word of your search term.
```
